.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/libsdl-org/SDL_image.git
+[submodule "lib/usb/ohci_usbhostlib"]
```

Comment:
*(This is a generic comment about PR description, but made on code so we can resolve it more easily)*

> However is still considered draft for these reasons:

Is it still draft? On GitHub it's no longer a draft PR